server/controllers/form.controller.js

```diff
@@ -1,8 +1,36 @@
-const getUserFormsController = async function (req, res) {
+const Form = require("../models/form.model");
+const Form = require("../models/form.model")
 
 
+const getUserFormsController = async function(req, res) {
+    try {
+        const { userId } = req.body
+        const usersFormsFromDb = await Form.find({})
```

Comment:
you're not using the userId to fetch forms, you're fetching all forms in the database

Reply:
` const usersFormsFromDb = await Form.find({ownerId:userId})`